apps/explorer/src/ui/TableCard.tsx

```diff
@@ -80,40 +66,79 @@ export function TableCard({ refetching, data, columns }: TableCardProps) {
-                                          )}
-                                </th>
-                            ))}
+                            {headerGroup.headers.map(
+                                ({
+                                    id,
+                                    colSpan,
+                                    column,
+                                    isPlaceholder,
+                                    getContext,
+                                }) => (
+                                    <th
+                                        key={id}
+                                        colSpan={colSpan}
+                                        scope="col"
+                                        className="h-7.5 px-1 text-left text-subtitle font-semibold uppercase text-steel-dark"
+                                        onClick={
+                                            column.columnDef.enableSorting
+                                                ? column.getToggleSortingHandler()
+                                                : void 0
+                                        }
+                                    >
+                                        <div
+                                            className={clsx(
+                                                'flex items-center gap-1',
+                                                column.columnDef
+                                                    .enableSorting &&
+                                                    'cursor-pointer text-steel-darker'
+                                            )}
+                                        >
+                                            {isPlaceholder
+                                                ? null
+                                                : flexRender(
+                                                      column.columnDef.header,
+                                                      getContext()
+                                                  )}
+                                            {{
+                                                asc: (
+                                                    <ArrowRight12
+                                                        fill="currentColor"
+                                                        className="-rotate-90 text-steel-darker"
+                                                    />
+                                                ),
+                                                desc: (
+                                                    <ArrowRight12
+                                                        fill="currentColor"
+                                                        className="rotate-90 text-steel-darker"
+                                                    />
+                                                ),
+                                            }[column.getIsSorted() as string] ??
```

Comment:
Just make a component <SortIcon /> that can take in a prop that's `asc` or `desc` and return the correct icon based on that. Will be easier to follow.